packages/ag-charts-community/src/chart/factory/processEnterpriseOptions.ts

```diff
@@ -86,3 +88,14 @@ export function removeUsedEnterpriseOptions<T extends Partial<AgChartOptions>>(o
         );
     }
 }
+
+export function removeUnusedEnterpriseOptions<T extends Partial<AgChartOptions>>(options: T) {
```

Comment:
This function prevents unused/disabled enterprise modules from being initialised downstream by completely removing their configuration.